demosplan/DemosPlanCoreBundle/Logic/OzgKeycloakUserDataMapper.php

```diff
@@ -218,17 +224,18 @@ private function detachUserFromOrgaAndDepartment(User $existingUser): void
      */
     private function updateOrganisation(Orga $existingOrga, array $requstedRoles): Orga
     {
+        //todo: shouldn't we throw an error instead!?
```

Comment:
leftover todo

Reply:
Just deleted the todo, because rework of the logic was not part of the ticket, therefore i guess this has a reason.